src/app/pages/swap/swap-summary/swap-summary.tsx

```diff
@@ -1,24 +1,24 @@
 import toast from 'react-hot-toast';
-import { FiCopy, FiExternalLink } from 'react-icons/fi';
+import { Outlet } from 'react-router-dom';
 
-import { Box, Stack, Text, useClipboard } from '@stacks/ui';
+import WaxSeal from '@assets/illustrations/wax-seal.png';
+import { useClipboard } from '@stacks/ui';
 import { useFormikContext } from 'formik';
+import { HStack, styled } from 'leather-styles/jsx';
 
 import { logger } from '@shared/logger';
 import { isUndefined } from '@shared/utils';
 
 import { useAnalytics } from '@app/common/hooks/analytics/use-analytics';
 // import { useExplorerLink } from '@app/common/hooks/use-explorer-link';
```

Comment:
Can you please delete this line now too? I know it was already like that

`// import { useExplorerLink } f..`

Reply:
These comments are still being left for devs working on other parts of this feature. I believe they can be removed soon with the alex PR I'm helping out with atm.